website/docs/setup/repos.md

```diff
@@ -159,3 +159,95 @@ need to use the following flag with the initial Corso `repo init` command:
 Corso also supports the use of object storage systems with no TLS certificate or with self-signed
 TLS certificates with the `--disable-tls` or `--disable-tls-verification` flags.
 [These flags](../../cli/corso-repo-init-s3) should only be used for testing.
+
+## Filesystem Storage
+
+Corso supports creating a repository on local or network attached filesystem.
+
+:::note
+Filesystem repositories aren't S3 compatible.
+:::
+
+### Filesystem storage prerequisites
+
+None.
+
+### Initialize filesystem repository
+
+Before first use, you need to initialize a Corso repository with `corso repo init filesystem`. See the command details
+[here](../../cli/corso-repo-init-filesystem). Corso will create the directory structure if necessary, including any
+missing parent directories.
+
+<Tabs groupId="os">
+<TabItem value="win" label="Powershell">
+
+  ```powershell
+  # Initialize the Corso Repository
+  $Env:CORSO_PASSPHRASE = 'CHANGE-ME-THIS-IS-INSECURE'
+  .\corso repo init filesystem --path C:\Users\user\corso-test
+  ```
+
+</TabItem>
+<TabItem value="unix" label="Linux/macOS">
+
+  ```bash
+  # Initialize the Corso Repository
+  export CORSO_PASSPHRASE="CHANGE-ME-THIS-IS-INSECURE"
+  ./corso repo init filesystem --path $HOME/corso-test
```

Comment:
General preference is for commands to look more like regular use and less like PoC values.  In this case, can we replace `-test` with some other suffix?  `-backup`?
```suggestion
  ./corso repo init filesystem --path $HOME/corso-test
```

Reply:
yeah, thats a great point. I noticed that we use `corso-test` in a few other places - [example](https://corsobackup.io/docs/quickstart/#create-a-corso-repository), was trying to stay consistent. I'll fix this as well as other occurrences.